app/assets/stylesheets/psique/foundation/_helpers.scss

```diff
@@ -36,3 +36,59 @@
 // -------------------------------------
 //   Animations
 // -------------------------------------
+
+/*
+   Animation example, for spinners
+*/
+.animate-spin {
+  -moz-animation: spin 2s infinite linear;
+  -o-animation: spin 2s infinite linear;
+  -webkit-animation: spin 2s infinite linear;
+  animation: spin 2s infinite linear;
```

Comment:
this smells as mixin.
`@include animation(spin 2s infinite linear);``
http://bourbon.io/docs/#animation